### ISSUE TITLE
Invalid set-cookie header

### ISSUE THREAD

**Issue by RutgerKe** [ubccr/mokey#12]
I tried to install mokey on the IPA server itself (CentOS), using a different URL to reach mokey and a wildcard certificate so both the freeipa admin interface and mokey can be reached using the same httpd server, so far so good. However, I am not able to login. The error returned by mokey is:

    level=error msg="tryauth: failed login attempt" ipa_client_error="ipa: login failed invalid set-cookie header" 

Googling the error I can trace it back [to a regex](https://github.com/ubccr/goipa/blob/master/ipa.go#L291), it seems like the `ipa_session` variable in the `Set-Cookie` header is empty. Which is strange, because testing it locally, doing a post request from my laptop to the server, works fine. And even on the server (where mokey is installed) a curl requests shows a valid session, but in the `IPASESSION` session header. 

Next step, the apache log, generated by ipa requests, I can see that another request is done before the requests to the `/ipa/session/login_password` url. One to `GET /ipa/session/cookie`.  If I understand it correctly, because the server is part of the same domain all requests to that domain are pre-authenticated by the server automatically. I even went as far as [logging the response headers](https://ian.io/2010/11/18/log-response-headers-in-apache.html). And indeed, the required Set-Cookie header is empty when doing a curl request from the server.

That is about where I got stuck. I thought of installing it on another server, but I read the requrement in the documentation 'mokey needs to be installed on a machine already enrolled in FreeIPA'. So I wonder, am I doing something wrong in my setup or is this a bug to be fixed?

Another note: The login credentials are correct, when I submit incorrect credentials on purpose I do get a different error in the log `ipa_client_error="IPA login failed with HTTP status code: 401" `. And when testing a user with 2FA it properly shows the 2FA login form.

Please let me know if you need more information, and thanks in advance

**Comment by aebruno** [comment]
@RutgerKe This is fixed in v0.0.6 but I have not released the rpm. I'm assuming you're running v0.0.5? I'll try and get an rpm build out soon. Apologies for the confusion.

**Comment by RutgerKe** [comment]
That's correct, I'm using the rpm version. Thanks for the quick reply! I'll await a new release.

**Comment by aebruno** [comment]
@RutgerKe Just released v0.0.6. This version includes some database changes so if you're upgrading and not starting fresh you'll need to run this command (after installing the new rpm):
```
mysql -u root -p mokey < /usr/share/mokey/ddl/upgrade-to-v0.0.6.sql
```

**Comment by RutgerKe** [comment]
Thanks, I did a fresh install and it works like a charm!
